README.md

```diff
@@ -1,13 +1,19 @@
 # XCP dissector
 
+![GitHub visitors](https://visitor-badge.laobi.icu/badge?page_id=835257451) ![GitHub stars](https://img.shields.io/github/stars/tracetronic/wireshark-xcp-dissector?style=flat) ![GitHub watchers](https://img.shields.io/github/watchers/tracetronic/wireshark-xcp-dissector?style=flat) ![GitHub release](https://img.shields.io/github/v/release/tracetronic/wireshark-xcp-dissector)
```

Comment:
Please add links with better targets to the badges (e.g. GitHub release should link to the Releases page)